applications/mne_scan/plugins/brainflowboard/README.md

```diff
@@ -0,0 +1,29 @@
+# BrainFlow Plugin
```

Comment:
This is a great guide! Maybe it would be a good idea to put it under our MNE-CPP 's docu website Develop > MNE Scan amplifiers section though. You can find a guide on how to improve the MNE-CPP docu [here](https://mne-cpp.github.io/pages/contribute/docuimprovements.html). It owuld also be helpful if you could include a description about the Plugins GUI.